ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1021,6 +1026,8 @@ class NoteTextComponent extends React.Component {
 				fn = this.commandShowLocalSearch;
 			} else if (command.name === 'textCode') {
 				fn = this.commandTextCode;
+			} else if (command.name === 'insertTemplate') {
+				fn = () => { return this.commandTemplate(command.value); };
```

Comment:
Could you bind commandTemplate to `this` (in the constructor) so that you can use the same syntax as the other calls?

Reply:
The reason for this syntax is because we need to pass command.value to the function. I changed the syntax of the call to fn, let me know what you think.